includes/class-wc-stripe-account.php

```diff
@@ -162,7 +162,7 @@ public function get_account_status() {
 		$settings_options = get_option( 'woocommerce_stripe_settings', [] );
 		$mode             = isset( $settings_options['testmode'] ) && 'yes' === $settings_options['testmode'] ? 'test' : 'live';
 
-		if ( empty( $account ) ) {
+		if ( empty( $account ) || isset( $account['errors'] ) ) {
```

Comment:
Does the account array always have a `errors` attribute? ie. could `$account['errors'] = []` or `$account['errors'] = ''`?

Reply:
When the request is successful there is only `account` data in response and the `errors` attribute is not there. But I think the code should be able to handle the cases you mentioned as well. Fixed in [this commit](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1969/commits/919331815635a31f7a7b66977e08bd8488976812)

Reply:
On second thought, we should actually avoid caching the error object. It was being cached before. Fixed [here](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1969/commits/6225152a8e4829378652df1a266f7c99f0de8643)